XamlControlsGallery/ControlPages/SliderPage.xaml

```diff
@@ -30,17 +31,53 @@
         <local:ControlExample x:Name="Example2" HeaderText="A Slider with range and steps specified.">
             <StackPanel x:Name="Control2" Orientation="Horizontal">
                 <Slider x:Name="Slider2" Header="Control header" Width="200" Minimum="500" Maximum="1000"
-                        LargeChange="100" StepFrequency="10" SmallChange="10" Value="800"/>
+                        LargeChange="100" StepFrequency="10" SmallChange="10" Value="800" Margin="0,0,10,0"/>
+                <Grid>
+                    <Grid.RowDefinitions>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                    </Grid.RowDefinitions>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto"/>
+                        <ColumnDefinition Width="Auto"/>
+                    </Grid.ColumnDefinitions>
+
+                    <TextBlock Text="Minimum:" Margin="0,5,0,0" Grid.Row="0" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="MinimumValue" Grid.Row="0" Grid.Column="1" Margin="10,5,0,0" Value="500" SpinButtonPlacementMode="Compact" ValueChanged="MinimumValue_ValueChanged"/>
+
+                    <TextBlock Text="Maximum:" Margin="0,5,0,0" Grid.Row="1" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="MaximumValue" Grid.Row="1" Grid.Column="1" Margin="10,5,0,0" Value="1000" SpinButtonPlacementMode="Compact" ValueChanged="MaximumValue_ValueChanged"/>
+
+                    <TextBlock Text="StepFrequency:" Margin="0,5,0,0" Grid.Row="2" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="StepFrequencyValue" Grid.Row="2" Grid.Column="1" Margin="10,5,0,0" Value="10" SpinButtonPlacementMode="Compact" ValueChanged="StepFrequencyValue_ValueChanged"/>
+
+                    <TextBlock Text="SmallChange:" Margin="0,5,0,0" Grid.Row="3" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="SmallChangeValue" Grid.Row="3" Grid.Column="1" Margin="10,5,0,0" Value="10" SpinButtonPlacementMode="Compact" ValueChanged="SmallChangeValue_ValueChanged"/>
+
+                    <TextBlock Text="LargeChange:" Margin="0,5,0,0" Grid.Row="4" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="LargeChangeValue" Grid.Row="4" Grid.Column="1" Margin="10,5,0,0" Value="100" SpinButtonPlacementMode="Compact" ValueChanged="LargeChangeValue_ValueChanged"/>
+                </Grid>
```

Comment:
I would place these values inside the ControlExample.Options part of the ControlExample. An example of this can be found here: https://github.com/microsoft/Xaml-Controls-Gallery/blob/master/XamlControlsGallery/ControlPages/NavigationViewPage.xaml#L231-L239

Reply:
> Thanks for that information.